client/components/views/Index.jsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { signIn, resetErrorLog, resetRedirect, resetLoadingState, verifyToken, googleLogin } from '../../actions';
+import SignInForm from './partials/SignInForm.jsx';
+import Footer from './partials/Footer.jsx';
+
+/**
+ * React component to display landing page
+ */
+class Index extends React.Component {
+  /**
+   * Component method called when component loads to reset state of spinner and hide side nav
+   * @returns {undefined} This method returns nothing
+   */
+  componentDidMount() {
+    this.props.resetLoadingState();
+    $('#sidenav-overlay').trigger('click');
+  }
+  /**
+   * Render method of React component
+   * @returns {Object} Returns the DOM object to be rendered
+   */
+  render() {
+    return (
+      <div>
+        <Body store={this.props}/>
+      </div>
+    );
+  }
+}
+
+/**
+ * React component that displays Navigation Bar
+ */
+class NavBar extends React.Component {
+  /**
+   * Render method of React component
+   * @returns {Object} Returns the DOM object to be rendered
+   */
+  render() {
+    return (
+      <div className="navbar-fixed">
+        <nav className="pink darken-4" role="navigation">
+          <div className="nav-wrapper">
+            <a href="#" id="brand" className="brand-logo">PostIt</a>
+            <a href="#" data-activates="mobile-demo"
+              className="button-collapse"><i className="material-icons">menu</i></a>
+            <ul className="right hide-on-med-and-down">
+              <li><a href="#">About PostIt</a></li>
+            </ul>
+            <ul id="mobile-demo" className="side-nav">
+              <li>
+                <div className="user-details">
+                  <div className="background">
+                    <img src="images/fire2.png" />
+                  </div>
+                </div>
+              </li>
+              <li><a href="#"><i className="large material-icons black-text">info</i>
+                About PostIt</a></li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+    );
+  }
+}
+
+
+/**
+ * React component for displaying page body
+ */
+class Body extends React.Component {
```

Comment:
for readability I think it would be nice to have `NavBar` and `Body` class in separate files and `import` it where you need it even if its just in one file